snippet/build_package.go

```diff
@@ -76,7 +76,7 @@ func BuildConn() (*net.TCPConn, error) {
 	return c, nil
 }
 
-func main() {
+func TestBuildPackage() {
```

Comment:
you can mv func TestBuildPackage into a new file build_package_test.go and use go Test/Example to test/demo your function.

https://golang.org/pkg/testing/

https://blog.golang.org/examples